Content.Client/Humanoid/HumanoidAppearanceSystem.cs

```diff
@@ -240,6 +284,7 @@ public void LoadProfile(EntityUid uid, HumanoidCharacterProfile profile, Humanoi
         }
     }
 
+    // TODO: Do marking things on server-side, so client only must apply it.
```

Comment:
since marking default coloring is now more complex instead of it just defaulting to skin color, maybe it would actually be nicer to just do the marking color setup on the server when the marking set is first populated instead of on the client at this point (i.e., just move some of this logic into `EnsureDefault` wrt fetching colors), this would also avoid the whole thing where you have to cache a hair color as well